### PR TITLE
Improve logging of connection maps.

### PR DESCRIPTION
**Pull request by nihohit**

### CODE REVIEW

File: redis/src/cluster_async/connections_container.rs [hunk 1]
```diff
@@ -28,6 +28,20 @@ where
 #[derive(Clone, Hash, Eq, PartialEq, Debug)]
 pub(crate) struct Identifier(IdentifierType);
 
+pub(crate) struct ConnectionsMap<Connection>(pub(crate) HashMap<ArcStr, ClusterNode<Connection>>);
+
+impl<Connection> std::fmt::Display for ConnectionsMap<Connection> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        for (key, value) in self.0.iter() {
```
Comment (barshaul) [inline redis/src/cluster_async/connections_container.rs:35]: can you rename key, value to identifier, node

File: redis/src/cluster_async/mod.rs [hunk 2]
```diff
@@ -562,36 +563,37 @@ where
...
-                (HashMap::with_capacity(initial_nodes.len()), None),
+                (
+                    ConnectionsMap(HashMap::with_capacity(initial_nodes.len())),
+                    None,
+                ),
                 |mut connections: (ConnectionMap<C>, Option<String>), addr_conn_res| async move {
                     match addr_conn_res {
                         Ok((addr, node)) => {
-                            connections.0.insert(addr.into(), node);
+                            connections.0 .0.insert(addr.into(), node);
```
Comment (barshaul) [inline redis/src/cluster_async/mod.rs:573]: Is this space intentionally?

File: redis/src/cluster_async/mod.rs [hunk 3]
```diff
@@ -562,36 +563,37 @@ where
...
                             (connections.0, None)
                         }
                         Err(e) => (connections.0, Some(e.to_string())),
                     }
                 },
             )
             .await;
-        if connections.0.is_empty() {
+        if connections.0 .0.is_empty() {
```
Comment (barshaul) [inline redis/src/cluster_async/mod.rs:581]: Same

File: redis/src/cluster_async/mod.rs [hunk 4]
```diff
@@ -562,36 +563,37 @@ where
...
             return Err(RedisError::from((
                 ErrorKind::IoError,
                 "Failed to create initial connections",
                 connections.1.unwrap_or("".to_string()),
             )));
         }
+        info!("create_initial_connections found nodes:\n{}", connections.0);
```
Comment (barshaul) [inline redis/src/cluster_async/mod.rs:588]: If it's in info logs, I think it shouldn't have the function name, something more general to the user, e.g.: "Found initial nodes {connections.0:?}"

**Review by barshaul — APPROVED** [review]

File: redis/src/cluster_async/mod.rs (hunk 2 above)
Comment (nihohit) [reply redis/src/cluster_async/mod.rs:573]: yup - autoformatted.

File: redis/src/cluster_async/mod.rs (hunk 3 above)
Comment (nihohit) [reply redis/src/cluster_async/mod.rs:581]: same

**Pull request merged**